src/libs/actions/Policy.js

```diff
@@ -143,14 +141,16 @@ function create(name = '') {
             Report.fetchChatReportsByIDs([response.policy.chatReportIDAdmins, response.policy.chatReportIDAnnounce, response.ownerPolicyExpenseChatID]);
 
             // We are awaiting this merge so that we can guarantee our policy is available to any React components connected to the policies collection before we navigate to a new route.
-            return Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${response.policyID}`, {
-                employeeList: getSimplifiedEmployeeList(response.policy.employeeList),
-                id: response.policyID,
-                type: response.policy.type,
-                name: response.policy.name,
-                role: CONST.POLICY.ROLE.ADMIN,
-                outputCurrency: response.policy.outputCurrency,
-            });
+            return Promise.all(
+                Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${response.policyID}`, {
+                    id: response.policyID,
+                    type: response.policy.type,
+                    name: response.policy.name,
+                    role: CONST.POLICY.ROLE.ADMIN,
+                    outputCurrency: response.policy.outputCurrency,
+                }),
+                Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY_MEMBER_LIST}${response.policyID}`, getSimplifiedEmployeeList(response.policy.employeeList)),
```

Comment:
Is it necessary to have both of these inside the promise.all? I don't think it matters when the member list is merged, does it? It also doesn't seem like it matters when the policy object is merged.

The only time the promise returned from `create()` is used is to navigate to the policy, but I don't think any of that requires the policy stuff to be merged into Onyx yet, does it?

Reply:
The policy data would need to be merged otherwise the workspace page would briefly open as "Not Found" until the Onyx update finishes. I included the members list here also just to be safe.

Reply:
OK, while I am fine not doing anything since it's only temporary, I do think it's important to always make clear and appropriate choices with code, regardless of how long it's going to live.